cli/pkg/cmd/install/root.go

```diff
@@ -29,12 +38,35 @@ func UninstallCmd(opts *options.Options, optionsFunc ...cliutils.OptionsFunc) *c
 		Short: constants.UNINSTALL_COMMAND.Short,
 		Long:  constants.UNINSTALL_COMMAND.Long,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := deleteNamespace(installNamespace); err != nil {
-				return errors.Wrapf(err, "delete sqoop failed")
+			fmt.Printf("Uninstalling Sqoop. This might take a while...")
```

Comment:
Should we maybe warn that we are deleting the entire namespace? I know we do this elsewhere as well, but it seems like people might have resources they don't want deleted.